transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -72,6 +75,16 @@ public Integer call() throws Exception {
             return EpollEventLoop.super.pendingTasks();
         }
     };
+
+    /**
+     * Metrics exposed to the user.
+     */
+    private final AtomicLong lastIoProcessingTime = new AtomicLong();
```

Comment:
Since these are for reporting could we use something with lower overhead like LongAdder?